src/main/java/com/casper/sdk/identifier/entity/AccountHashEntityIdentifier.java

```diff
@@ -0,0 +1,23 @@
+package com.casper.sdk.identifier.entity;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.*;
+
+/**
+ * The account hash of an account.
+ *
+ * @author carl@stormeye.co.uk
+ */
+@Getter
+@Setter
+@Builder
+@AllArgsConstructor
+public class AccountHashEntityIdentifier implements EntityIdentifier{
```

Comment:
Please format your classes before pushing. Should be whitespace after interface name. Applies to all classes implementing EntityIdentifier.